source/_changelogs/5.2.0.md

```diff
@@ -0,0 +1,39 @@
+# 5.2.0
+
+*Released 9/14/2020*
+
+**Features:**
+
+- Added the configuration option `includeShadowDom` for enabling shadom DOM querying globally, per-suite, per-test, or programmatically. Addresses {% issue 8442 %}.
```

Comment:
@chrisbreiding - small typo here:

"for enabling shadom DOM" should be "for enabling shadow DOM"

Reply:
Good catch - thanks!